src/bots/depositor.py

```diff
@@ -147,23 +141,23 @@ def _check_balance(self):
                 GUARDIAN_BALANCE.labels(address=address, chain_id=w3_databus_chain_id).set(balance)
 
     def _deposit_to_module(self, module_id: int) -> bool:
-        is_depositable = self._check_module_status(module_id)
-        logger.info({'msg': 'Fetch module depositable status.', 'value': is_depositable})
-
-        quorum = self._get_quorum(module_id)
-        logger.info({'msg': 'Build quorum.', 'value': quorum})
+        if module_id not in variables.DEPOSIT_MODULES_WHITELIST:
```

Comment:
I think we can remove this check. Since we are filtering such modules here
`_get_preferred_to_deposit_modules`

Reply:
Done